src/object/cli_ec.c

```diff
@@ -2806,7 +2806,6 @@ obj_recx_ec_daos2shard(struct daos_oclass_attr *oca, int shard,
 
 	if (total == 0) {
 		D_FREE(*recxs_p);
-		*recxs_p = NULL;
```

Comment:
Why removing this line?

Reply:
oh, D_FREE() will reset *resxs_p to NULL anyway. Xuezhao point this out in my last patch. so I update it in  this patch, sorry.

Reply:
Ah, I didn't know D_FREE modifies the pointer. Thanks.